bin/rucio

```diff
@@ -38,11 +38,9 @@ def map_legacy_command() -> Optional[list[str]]:
     except IndexError:
         command = None
 
-    new_command = None
```

Comment:
The removal of this line is causing errors in the CI due of variable usage before assignment